### DISCUSSION TITLE
整理刮削生产的nfo文件，语言混杂，同时出现中英文描述nfo文件

### DISCUSSION THREAD

**Discussion by chux1995** [jxxghp/MoviePilot#3352] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

v1.8.9

### 运行环境

Docker

### 问题类型

主程序运行问题

### 问题描述

附上刮削完成的其中一个电视剧的nfo文件内容，整个剧集的描述都是中文，但是单个剧集有的就是英文，实际上tmdb中都是有中文剧集描述，怀疑是整理逻辑出现bug。虽然可以使用emby重新刮削可以解决，但是以后每个剧集更新一集都得刮削一遍实在是太烦人了。
<img width="1078" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/36070604/04361952-3783-4de1-89cc-c05ce2ab8137">
<img width="1098" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/36070604/d987ae3d-4efd-4e38-ab2d-81e15d7d877e">
<img width="1673" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/36070604/99c24d6f-c2cc-449a-9503-5429f53b9058">
<img width="1640" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/36070604/7463e031-ccc6-4e07-aa7f-95b087574840">

**Comment by chux1995** [comment]
还有一点就是，我目前在使用infuse播放器，直接扫描link文件目录的配置方式，目前遇见会与本地nfo文件识别发生冲突，视频的识别日期错误，邮件得到的反馈如图。因为infuse本身的命名识别成功率挺高的，就是可以提供一个开关，不生产nfo文件吗，单纯只进行文件整理重命名。
<img width="534" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/36070604/ecdce9d2-1b5e-43c1-aed7-2b8edebd6e82">

**Comment by jxxghp** [comment]
1. 时间差问题吧？刮消时还没有中文信息，后面才有的，此时不会重新生成，所以才有一些插件是通知定时媒体服务器刷新最近入库信息功能。
2. 这个开关已经有了。

**Comment by chux1995** [comment]
对的，系统设置有开关，我找到了。不过这个视频是2022年的，之前我刮削过中文的，不会是后来更新的这个原因，而且这里的情况是，剧集总概nfo是中文，但是剧集的nfo是英文。

**Reply by liuyuze2005** [reply]
元数据提供者顺序

确保：

TMDB 在前
TVDB 不要优先

因为 TVDB 很多剧集中文不完整，会回退英文。

**Comment by comozen** [comment]
同出现过这个中英混杂的问题
